firefox_puppeteer/ui/toolbars.py

```diff
@@ -436,6 +436,12 @@ def close(self, force=False):
 
         Wait(self.marionette).until(lambda _: not self.is_open)
 
+    def open(self):
+        """Opens the identity popup
+        """
+        self.locationbar.identity_box.click()
```

Comment:
`locationbar` is not a property of the IdentityPopup class. What you may want is to add the method to the LocationBar class as `open_identity_popup()`.